src/main/java/com/boot/cart/controller/CartController.java

```diff
@@ -60,28 +60,28 @@ public ResponseEntity<CartDTO> removeProductfromCart(@Email(message = "Invalid e
         return new ResponseEntity<>(newCart, HttpStatus.OK);
     }
 
-    @DeleteMapping("/deleteCartByEmail")
+    @DeleteMapping
     public ResponseEntity<CartDTO> deleteCartByEmail(@Email(message = "Invalid email!", regexp = Constants.EMAIL_REGEXP) @RequestParam("email") String email)
             throws EntityNotFoundException {
         cartService.deleteCartByEmail(email);
         return new ResponseEntity<>(HttpStatus.OK);
     }
 
-    @GetMapping("/getCartByEmail")
```

Comment:
if you use /cart/{email} you could keep GET /cart for get all endpoint

Reply:
done!